### ISSUE TITLE
Fail to create zim

### ISSUE THREAD

**Issue by project21212** [openzim/wp1#779]
I have tried creating a zim for the following projects, each time I check back when it is done, creating of the zim has failed.
Medicine
Society and medicine

**Comment by audiodude** [comment]
Yes there seems to be a problem with the latest version of mwoffliner, our sister project. Log is here for a run I made to try and reproduce:

https://api.farm.zimit.kiwix.org/v1/tasks/d387121f-81a0-4350-b0e9-591d6cc3afeb

Will move this issue there.
